File: app/src/main/java/com/example/pearvideoclient/utils/CollectionUtil.java
package com.example.pearvideoclient.utils;

import java.util.Collection;

/**
 * @Description: java类作用描述
 * @Author: jiazhu
 * @Date: 2019-08-31 09:23
 * @ClassName: CollectionUtil
 */
public class CollectionUtil {
    public static boolean isEmpty(Collection<?> collection) {
        return collection == null || collection.isEmpty();
    }
}

File: app/src/main/java/com/example/pearvideoclient/utils/CollectionUtil.kt
package com.example.pearvideoclient.utils

/**
 * @Description: java类作用描述
 * @Author: jiazhu
 * @Date: 2019-08-31 09:23
 * @ClassName: CollectionUtil
 */
object CollectionUtil {
    @JvmStatic
    fun isEmpty(collection: Collection<*>?): Boolean {
        return collection == null || collection.isEmpty()
    }
}
